docs/future-work/oss-communication-improvements/build-a-tutorial.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Build a Tutorial
+rfc: not-required
+estimated-scope: medium
+improved-metric: community-engagement
+skills: video-production, writing, python
+output-type: tutorial
+status: evergreen
+---
+
+Create a tutorial with your experience with Monty to accomplish some goal.
+
+## Format Options
+
+The tutorial can be created in either format:
+- YouTube video
+- Blog post
+
+Both formats may potentially become core documentation if they add value to the existing getting started guides.
+
+
+## Potential Tutorial Topics
```

Comment:
I think it would be good to state that the best way to find a topic is to write about something that was hard to learn about where the person wished they had a tutorial about.